build_test/lib/src/copy_builder.dart

```diff
@@ -59,13 +55,8 @@ class CopyBuilder implements Builder {
     return {'': outputs};
   }
 
-  AssetId _copiedAssetId(AssetId inputId, int copyNum) {
-    var withExtension = inputId.addExtension(_copiedAssetExtension(copyNum));
-    if (outputPackage == null) {
-      return withExtension;
-    }
-    return new AssetId(outputPackage, withExtension.path);
```

Comment:
This is the breakage I was calling out here: https://github.com/dart-lang/build/issues/222#issuecomment-295940412